api/src/main/java/jakarta/data/page/CursoredPage.java

```diff
@@ -154,7 +154,7 @@ public interface CursoredPage<T> extends Page<T> {
      * @param index position (0 is first) of a result on the page.
      * @return cursor for key values at the specified position.
      */
-    PageRequest.Cursor getCursor(int index);
+    PageRequest.Cursor cursor(int index);
```

Comment:
This is not a record style accessor. It accepts a parameter value.

Reply:
That is a good point, but it still accesses info.

IMHO: It should keep without "get" as the other ones.